src/application/components/Layouts/__tests__/Navigation.test.tsx

```diff
@@ -19,19 +19,22 @@ describe("<Navigation />", () => {
   });
 
   it("renders the navigation items", () => {
-    const { getByText } = renderWithApolloClient(<Navigation {...props} />);
-    expect(getByText("Explorer")).toBeInTheDocument();
-    expect(getByText(`Queries (${props.queriesCount})`)).toBeInTheDocument();
-    expect(getByText(`Mutations (${props.queriesCount})`)).toBeInTheDocument();
-    expect(getByText("Cache")).toBeInTheDocument();
+    renderWithApolloClient(<Navigation {...props} />);
+
+    expect(screen.getByText("Explorer")).toBeInTheDocument();
+    expect(screen.getByText(`Queries (${props.queriesCount})`)).toBeInTheDocument();
+    expect(screen.getByText(`Mutations (${props.queriesCount})`)).toBeInTheDocument();
+    expect(screen.getByText("Cache")).toBeInTheDocument();
   });
 
-  it("can select a navigation item", () => {
-    const { container } = renderWithApolloClient(<Navigation {...props} />);
+  it("can select a navigation item", async () => {
+    const { container, user } = renderWithApolloClient(<Navigation {...props} />);
     const buttons = container.querySelectorAll("button");
     const lastButton = buttons[buttons.length - 1];
     expect(lastButton).not.toHaveStyleRule("color", colors.silver.lighter);
-    user.click(lastButton);
+    await act(async () => {
+      await user.click(lastButton);
+    });
```

Comment:
```suggestion
    await act(() => user.click(lastButton));
```
This already returns a promise, no need to double-await :)

Reply:
Good catch! Updated in [0f71fc5](https://github.com/apollographql/apollo-client-devtools/pull/994/commits/0f71fc56f4b281ccb604b6ce7e42a97c93246b66)